src/app/components/Balances/index.tsx

```diff
@@ -0,0 +1,217 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Tooltip, Progress, Tabs, Icon, Select, Alert } from 'antd';
+import Statistic from 'antd/lib/statistic'; 
```

Comment:
Very cool, did not know about this component!

Reply:
Yeah, I discovered it by looking at the antd pro demo site